sympy/functions/special/gamma_functions.py

```diff
@@ -701,7 +701,7 @@ def _eval_expand_func(self, **hints):
                         return Add(*tail)/coeff**(n + 1)
                 z *= coeff
 
-        if n == 0 and z.is_rational and z.is_Number:
+        if n == 0 and z.is_rational:
```

Comment:
I'm sorry if I didn't make it clear: this should be  `is_Rational`, not `is_rational`. These are different checks, like `is_Integer` vs `is_integer` or `is_Number` vs `is_number`. 

`is_Rational` checks that the object is a concrete rational number. `is_rational` checks whether it can be inferred that the object represents some rational number. For example `Symbol('x', rational=True)` passes the latter but not the former.